code/script.js

```diff
@@ -1,51 +1,154 @@
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
-}
+};
 
 // Starts here
-const greetUser = () => {
+
+//Greeting
+const greeting = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage("Hello, enter your name and table number.", "bot");
+  nameForm.addEventListener("submit", handleInput);
+};
+
+
+//User answering his name
+const handleInput = (event) => {
+  event.preventDefault(); //prevents refreshing the page
+  let userName = nameInput.value;
+  showMessage(`Name and table number: ${userName}.`, "user");
+  nameInput.value = ""; //clears the answer field
+  setTimeout(() => question1(userName), 2000);  // goes to next question, and passes username as a prop. 
+};
+
+//Bot askes about the order
+const question1 = (userName) => {
+  showMessage(`Name and table number: ${userName}. Thank you. Are you over 20 years old?`, "bot");
+  setTimeout(() => question2(), 1000);
+};
+
+
+//Bot asking if user is over 20
+const question2 = (event) => {
+  inputWrapper.innerHTML = `
+  <button id="yes" type="submit" class="chat-btn">Yes</button>
+  <button id="no" type="submit" class="chat-btn">No</button>
+  `
+  // when a button is clicked, an event listener "listens" to the event and invoked the corresponding function
+  document.getElementById("yes").addEventListener("click", yesChoice);  
+  document.getElementById("no").addEventListener("click", noChoice);
+};
+
+//If user says no, I changes the question3nochoice to 2 so next question doesnt appear
+const noChoice = (event) => {
+  event.preventDefault();
+  let noChoice = inputWrapper.value;
+  showMessage("Sorry underaged, cannot order wine", "user");
+  inputWrapper.value = "";
+  setTimeout(() => question2(noChoice), 1000);
+};
+
+// if user is over 20 yrs
+const yesChoice = (event) => {
+  event.preventDefault(); 
+  let yesChoice = inputWrapper.value;
+  showMessage("Over 20", "user");
+  inputWrapper.value = "";
+  setTimeout(() => question3(yesChoice), 2000);
+};
+
+
+//Bot asking red or white user wants to order
+const question3 = (yesChoice) => {
+  showMessage("Good, you are over 20! What wine do you prefer?", "bot");
+  inputWrapper.innerHTML = `
+  <button id="white" type="submit" class="chat-btn">White wine</button>
+  <button id="red" type="submit" class="chat-btn">Red wine</button>
+  `
+  document.getElementById("white").addEventListener("click", whiteWine);
+  document.getElementById("red").addEventListener("click", redWine);
+};
+
+ 
+// red or white choice
+const whiteWine = (event) => {
```

Comment:
Jag hittade i sidan som technigo länkade ett bra knep med lista som gick att änvända om man inte vill ha endast knappar som alternativ utöver siffror och text:
 if (type === 'yes') {

     showMessage ('input your bot reply here')

    inputWrapper.innerHTML = `
      <select id="select">
        <option value="" selected disabled>Tell me of...</option> // låter di ha ett förvalt alt. när menyn dyker upp
        <option value="chablis">chablis</option>
        <option value="pinot">pinot noir</option>
        <option value="somegrape">somegrape</option>
      </select>
    `
  } else {  osv....
  
  var ett roligt alternativ som var kul att laborera med utöver knappar och text!

Reply:
Ja precis bra tänkt, i min urspungliga ide så hade jag tänkt ha olika vinsorter som nästa alternativ men det blev lite för svårt för mig så jag skippade det. Får se om jag löser det nån gång :)